data/test/scenarios/wml_tests/ScenarioWML/EventWML/events-test_filter_ability.cfg

```diff
@@ -296,3 +303,140 @@
 )}
 
 #undef FILTER_ABILITY_TEST
+
+##
+# Common code for the tests in this file.
+##
+# Actions:
+# Give both Alice and Bob 100% chance to hit.
+# Give Alice a drains ability which is only active during liminal times of day and to value 25(defined in hardcoding by default).
```

Comment:
This line's documentation about the value is wrong.

Reply:
fixed

Reply:
It's also not locked to liminal times of day.

Reply:
fixed